app/db/models.py

```diff
@@ -1,54 +1,110 @@
 """ Class models that reflect resources and map to database tables
 """
 
-from sqlalchemy import Column, String, Integer, Float, DateTime, Sequence, ForeignKey
-from sqlalchemy.sql import func
-from db.database import BASE
+from sqlalchemy import (Column, String, Integer, Float,
+                        TIMESTAMP, Sequence, ForeignKey, UniqueConstraint)
+from sqlalchemy.orm import relationship
+from sqlalchemy.dialects.postgresql import ARRAY
+from geoalchemy2 import Geometry
+from app.db.database import Base
 
 
-class ForecastModel(BASE):
-    """ Weather forecasting model blueprint """
-    __tablename__ = 'forecast_model'
+class ProcessedModelRunFile(Base):
```

Comment:
Can we add descriptions for each table?
looks like it can be achieved by this
https://stackoverflow.com/questions/38709800/how-to-add-comments-to-table-or-column-in-mysql-using-sqlalchemy